### PR TITLE
Added Contribution Guidelines

### PR DESCRIPTION
**Pull request by anthony-ngu**

### CODE REVIEW

File: ContributionGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+permalink: /ContributionGuidelines.htm
+---
+
+# Contribution Guidelines
+
+___
+
+**Maintainers** – (noun) – the people managing the merging of pull requests and the state of the GitHub repositories.
+
+### Coding Guidelines:
```
Comment (ooeygui) [inline ContributionGuidelines.md:11]: This doesn't work if you don't have a double hash as a parent.

File: ContributionGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+layout: code
+title: Contribution Guidelines
+permalink: /ContributionGuidelines.htm
+---
+
+# Contribution Guidelines
+
+___
+
+**Maintainers** – (noun) – the people managing the merging of pull requests and the state of the GitHub repositories.
```
Comment (ooeygui) [inline ContributionGuidelines.md]: Cute, but I don't know if that's appropriate.

Comment (anthony-ngu) [inline ContributionGuidelines.md]: The maintainers blurb?

File: ContributionGuidelines.md (hunk 1 above)
Comment (anthony-ngu) [inline ContributionGuidelines.md:11]: What do you mean by "double hash as a parent"?

File: ContributionGuidelines.md (hunk 2 above)
Comment (ooeygui) [inline ContributionGuidelines.md]: yes.

File: ContributionGuidelines.md (hunk 1 above)
Comment (ooeygui) [inline ContributionGuidelines.md:11]: # Title
 ### SubSubTitle

Doesn't work.
You need:
 # Title
 ## SubTitle
 ### SubSubTitle

Comment (anthony-ngu) [inline ContributionGuidelines.md:11]: It works.. There is no need to parent subtitles.

File: ContributionGuidelines.md (hunk 2 above)
Comment (anthony-ngu) [inline ContributionGuidelines.md]: got rid of it :)

**Pull request merged**